src/js/utils/ServiceUtil.js

```diff
@@ -0,0 +1,53 @@
+import Service from '../structs/Service';
+
+const getFindPropertiesRecursive = function (service) {
```

Comment:
Is there any special reason you curry this? Could be a plain function with to params...